### PR TITLE
Create a main file

### PR DESCRIPTION
**Pull request by amexabee**
- Create a main.rb file that will serve as your console app entry-point
- Present the user with a list of options to perform.
- Let users choose an option.
- Have a way to quit the app.

### CODE REVIEW

File: src/main.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  loop do
+    puts "\nPlease choose an option by entering a number
+    1. Get genre
+    2. Get author
+    3. Get source
+    4. Get label
+    5. Get published date
+    6. Exit"
+    input = gets.chomp.to_i
```
Comment (mperezbazan) [inline src/main.rb:12]: - [ ] Kindly complete the options that are shown in the project requirements
image

**Review by mperezbazan — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @amexabee 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/main.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    8. List all sources (e.g. 'From a friend', 'Online shop')
+    9. Add a book
+    10. Add a music album
+    11. Add a movie
+    12. Add a game"
+
+    if gets.chomp.to_i == 6
+      puts "\nGood Bye."
+      break
+    end
```
Comment (mperezbazan) [inline src/main.rb:22]: - [] Kindly check to put an option to Exit for the application, in this moments you can exit with number 6 but the option is "List all labels"

**Review by mperezbazan — CHANGES_REQUESTED** [review]
Hi @amexabee  , 

Great work on making the changes requested by a previous review 👏 
You've done well implementing some of the requested changes, but there are still some which you have to work on it


 ## Highlights
✔️ Github flow 
✔️ PRO PR
✔️ No linter error 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/main.rb (hunk 2 above)
Comment (amexabee) [reply src/main.rb]: Thank you. I forgot to update that.

**Pull request merged**